.travis.yml

```diff
@@ -33,10 +33,6 @@ matrix:
   exclude:
   # Only test Rails6 on supported rubies
   # Don't test lowest rails support on maintained rubies
-    - rvm: 2.3
-      gemfile: gemfiles/rails_6_0.gemfile
-    - rvm: 2.4
-      gemfile: gemfiles/rails_4_2.gemfile
     - rvm: 2.4
       gemfile: gemfiles/rails_6_0.gemfile
     - rvm: 2.5
```

Comment:
I think tests should run for ruby 2.5 and rails 4.2 combo as well

Reply:
I agree but tried to go keep with the spirit of the set of exclusions that was already there.

Perhaps @luke-hill, who created this set in the first place, can explain the reasoning for not testing 4.2 on Ruby 2.5?

Reply:
At the time we originally drafted the 2.5 addition to travis (Or at least first started culling the travis releases), we made a conscious decision that the "newest" ruby at the time wouldn't support the oldest rails we supported. Because that Max/Min combo was perceived to be not desirable.

Also 2.5 (For another reason), was a Rails6 enforcement. So when we introduced rails6, it felt like most people on 2.5 would go straight to rails6 for big gains.

Reply:
Right, so Rails 4.2 would only be tested with Ruby 2.4. This makes sense to me.

I'm going to mull over the possibility of inverting the logic here, because with so many exclusions I find it hard to have a clear overview of what will be built. I don't know if that's possible with Travis' config, but I'll see.

Reply:
My philosophy here is if you can justify the change that's fine. I'm not gonna rule with an iron fist here.

By default (With simple maths), we have 5*5 tested permutations. And I think Rails6 rules out the top2/3. So realistically by default we have like 22 permutations.

We need to get that down a little bit, but then I suppose it's also about volume. If we're not really firing many PR's at it, I'm happy to open it up a little bit, I just wouldn't want something previously excluded to be "reincluded" so to speak.